ksql-cli/src/main/java/io/confluent/ksql/cli/commands/AbstractCliCommands.java

```diff
@@ -84,6 +84,8 @@ public void run() {
       } else {
         cli.runInteractively();
       }
+    } catch (RuntimeException exception) {
```

Comment:
Any reason we need the extra catch block?

Reply:
Its for filtering out RuntimeExceptions from the next catch block. If they get caught in the catch(Exception exception) block we re-wrap them in RuntimeException and then print a redundant line in the final error message.

Reply:
Presumably to avoid wrapping a `RuntimeException` inside another `RuntimeException` which is what would happen if this catch wasn't in place.